src/main/java/com/iotiq/application/domain/MaterialParameter.java

```diff
@@ -17,16 +17,16 @@
 @AllArgsConstructor
 @NoArgsConstructor
 public class MaterialParameter {
-    @DecimalMin("0.05")
-    @DecimalMax("2.00")
+    @DecimalMin(value = "0.05", message = "{exception.greaterThan}")
```

Comment:
thickness and weightPerUnitArea are both Float but validated using different annotations. Why is that?

Reply:
@Min annotations support float values, but to set a minimum value constraint of 0.05, I need to use @DecimalMin.